packages/ssr/README.md

```diff
@@ -0,0 +1,49 @@
+# <a href="https://www.apollographql.com/"><img src="https://user-images.githubusercontent.com/841294/53402609-b97a2180-39ba-11e9-8100-812bab86357c.png" height="100" alt="React Apollo"></a>
+
+## React Apollo - Server-Side Rendering
+
+[![npm version](https://badge.fury.io/js/%40apollo%2Freact-ssr.svg)](https://badge.fury.io/js/%40apollo%2Freact-ssr)
+[![Build Status](https://circleci.com/gh/apollographql/react-apollo.svg?style=svg)](https://circleci.com/gh/apollographql/react-apollo)
+[![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/apollo)
+
+### Contents
+
+1. [Installation](#installation)
+2. [Overview](#overview)
+3. [Reference](#reference)
+
+### Installation
+
+```
+npm install @apollo/react-ssr
+```
+
+### Overview
+
+<a name="getDataFromTree"></a>
+#### a) [`getDataFromTree`](https://github.com/apollographql/react-apollo/blob/release-3.0.0/packages/ssr/src/getDataFromTree.ts)
```

Comment:
This `README.md` needs to be fleshed out before this PR is merged (and before this package is published).

Reply:
I'm going to do a full review/edit on all RA 3 package `README`'s before publishing, so feel free to merge this PR as is for now.